pxr/base/arch/debugger.cpp

```diff
@@ -342,50 +343,59 @@ Arch_DebuggerAttachExecPosix(void* data)
 
 #if defined(ARCH_OS_LINUX)
 
+// Reads /proc/self/status, finds the line starting with "field:", and
+// returns the portion following the ":".
+// Note that the returned string will generally include leading whitespace
 static
-bool
-Arch_DebuggerIsAttachedPosix()
+std::string Arch_ReadProcStatusField(const char* field)
```

Comment:
If you use `std::string_view` here, you don't need to use `strlen` or `strcmp` and don't have to worry about potentially handling `nullptr` `field` values.

Reply:
Good point - done: https://github.com/PixarAnimationStudios/OpenUSD/pull/3014/commits/ed2b39b4d6a428db954ed3dae5153cfb8a618e72